doc/nrf/known_issues.rst

```diff
@@ -360,6 +360,19 @@ Subsystems
 Bluetooth LE
 ============
 
+.. rst-class:: v1-5-0 v1-4-2 v1-4-1 v1-4-0
+
+DRGN-15435: GATT Notifications and Write without Responses can be sent out of order.
+  GATT Notifications and Write without Responses can be sent out of order when not using a complete callback.
+
+  **Workaround:** Always set a callback for Notifications and Write without Response.
```

Comment:
```suggestion
DRGN-15435: GATT notifications and Write without Response might be sent out of order
  GATT notifications and Write without Response might be sent out of order when not using a complete callback.

  **Workaround:** Always set a callback for notifications and Write without Response.
```
Could also use "Writes without Response" or "Write without Response commands" if plural is crucial, but I think it works fine without.

Reply:
Why remove the capitalization of Notifications, but not Write Without Response?
Plural is not crucial, but remove plural on Notifications as well then, right?

Reply:
Because notification is a normal word and can be read as just the concept, not the procedure, which is easier to grasp. While Write Without Response only works as set term. (Actually, without should be capitalized as well, missed that in the suggestion.)
Regarding the plural - if I understand it correctly, we're not talking about exactly one notification and one Write, so the most consistent way of writing it is "GATT notifications and Write without Response commands" imo. But trying to save some words. ;-) If we leave out the plural for notifications, I would read this as talking about one notification and one write.

Reply:
How about GATT notifications and Writes Without Response?

I'd rather not use the term "command" as it blurs the line between the protocols (Write Without Response in GATT, Write Command in ATT).

Strictly speaking it can happen to just one Notification or Write, being reordered with whatever else is queued. But it would probably only happen or be noticeable when having multiple notifications or writes.

Reply:
Sure, Writes Without Response is okay, as suggested above.